x-pack/plugins/security_solution/public/overview/components/detection_response/alerts_by_status/index.tsx

```diff
@@ -0,0 +1,226 @@
+  const CasesContext = cases.ui.getCasesContext();
+  const userPermissions = useGetUserCasesPermissions();
+  const userCanCrud = userPermissions?.crud ?? false;
+  const { colors } = useContext(ThemeContext);
+  const { toggleStatus, setToggleStatus } = useQueryToggle(queryId);
+
+  const { items: donutData, isLoading: loading } = useAlertsByStatus({
+    signalIndexName,
+    queryId,
+    skip: !toggleStatus,
+  });
+  const legendItems: LegendItem[] = useMemo(
+    () =>
+      donutData && donutData?.length > 0 && legendField
+        ? (donutData[0] && donutData[0].buckets).map((d, i) => ({
+            color: colors[d.key],
+            dataProviderId: escapeDataProviderId(`draggable-legend-item-${uuid.v4()}-${d.key}`),
+            timelineId: undefined,
+            field: legendField,
+            value: d.key,
+          }))
+        : NO_LEGEND_DATA,
+    [colors, donutData]
+  );
+
+  const totalAlerts = useMemo(
+    () => donutData.reduce((acc, curr) => acc + curr.doc_count, 0),
+    [donutData]
+  );
+
+  const fillColor = useCallback(
+    (d: Datum) => {
+      switch (d.dataName) {
+        case 'low':
+          return colors.low;
+        case 'medium':
+          return colors.medium;
+        case 'high':
+          return colors.high;
+        case 'critical':
+          return colors.critical;
+        default:
+          return colors.low;
+      }
+    },
+    [colors.critical, colors.high, colors.low, colors.medium]
+  );
+  return (
+    <>
+      <HoverVisibilityContainer show={true} targetClassNames={[HISTOGRAM_ACTIONS_BUTTON_CLASS]}>
+        <HistogramPanel
+          data-test-subj={`${queryId}Panel`}
+          height={toggleStatus ? 'auto' : panelSettings.panelHeight}
+          paddingSize={panelSettings.paddingSize}
+        >
+          {loading && (
+            <EuiProgress
+              data-test-subj="initialLoadingPanelMatrixOverTime"
+              size="xs"
+              position="absolute"
+              color="accent"
+            />
+          )}
+          <HeaderSection
+            id={queryId}
+            title={title}
+            subtitle={subtitle}
+            inspectMultiple
+            toggleStatus={toggleStatus}
+            toggleQuery={setToggleStatus}
+          >
+            <EuiFlexGroup alignItems="center" gutterSize="none">
+              {visualizationActionsOptions && (
+                <EuiFlexItem grow={false}>
+                  <CasesContext owner={[APP_ID]} userCanCrud={userCanCrud ?? false}>
+                    <VisualizationActions
+                      {...visualizationActionsOptions}
+                      className={classnames(
+                        visualizationActionsOptions.className,
+                        'histogram-viz-actions'
+                      )}
+                      isInspectButtonDisabled={false}
+                      queryId={queryId}
+                      title={title}
+                    />
+                  </CasesContext>
```

Comment:
the `visualizationActionsOptions` are never passed, will they be used in this version? If we are not going to be used in this version it is better to just remove the code, we will add it later if needed.